packages/cli/.vscode/launch.json

```diff
@@ -1,21 +1,14 @@
 {
```

Comment:
the debug launch configuration was added per package, in case a single package folder is opened in VSCode to work on